.travis.yml

```diff
@@ -24,28 +24,26 @@ matrix:
     - php: 5.4
       env: SYMFONY_VERSION=2.8.*
     - php: 5.6
-      env: SYMFONY_VERSION=2.8.* CS_FIXER=run
+      env: SYMFONY_VERSION=2.8.* SYMFONY_DEPRECATIONS_HELPER=strict CS_FIXER=run
```

Comment:
Done.  Also renamed the two tests testing legacy behavior in the bundle to silence the uncaught deprecation warnings.